tests/test-loader.js

```diff
@@ -3,5 +3,11 @@
 // See: https://github.com/webpack/karma-webpack#alternative-usage
 import 'babel-core/polyfill';
 
+// silence logging in components
+var logLevels = ['log', 'warn', 'info', 'error'];
```

Comment:
What about `debug`? It obviously doesn't get used but adding it would be future-proof.